src/h5web/toolbar/controls/ColorMapSelector.tsx

```diff
@@ -37,53 +36,15 @@ interface Props {
 function ColorMapSelector(props: Props): ReactElement {
   const { value, disabled, onChange } = props;
 
-  const { height: winHeight } = useWindowSize();
-  const menuMaxHeight = winHeight - MENU_TOP - MENU_BOTTOM;
-
   return (
-    <Wrapper className={styles.wrapper} onSelection={onChange}>
-      <Button className={styles.btn} tag="button" disabled={disabled}>
-        <div className={styles.btnLike}>
-          <span className={styles.selectedColorMap}>
-            {value}
-            <Gradient interpolator={INTERPOLATORS[value]} />
-          </span>
-          <MdArrowDropDown className={styles.icon} />
-        </div>
-      </Button>
-
-      <Menu
-        className={styles.menu}
-        style={{ maxHeight: Math.min(MENU_IDEAL_HEIGHT, menuMaxHeight) }}
-      >
-        <ul className={styles.list}>
-          {Object.entries(INTERPOLATOR_GROUPS).map(
-            ([groupLabel, interpolators]) => (
-              <li key={groupLabel}>
-                <span className={styles.groupLabel}>{groupLabel}</span>
-                <ul className={styles.list}>
-                  {Object.entries(interpolators).map(
-                    ([colorMap, interpolator]) => (
-                      <li key={colorMap}>
-                        <MenuItem
-                          className={styles.option}
-                          text={colorMap}
-                          value={colorMap}
-                          data-selected={colorMap === value || undefined}
-                        >
-                          {colorMap}
-                          <Gradient interpolator={interpolator} />
-                        </MenuItem>
-                      </li>
-                    )
-                  )}
-                </ul>
-              </li>
-            )
-          )}
-        </ul>
-      </Menu>
-    </Wrapper>
+    <Selector
+      value={value}
+      disabled={disabled}
+      onChange={onChange}
+      options={COLORMAP_GROUPS}
+      selectedComponent={ColormapOption}
+      optionComponent={ColormapOption}
```

Comment:
It seems that `selectedComponent` and `optionComponent` are always the same, in the end. Is it worth keeping them as separate props?